src/components/Experiment/QuickInput/QuickInput.stories.js

```diff
@@ -89,3 +90,26 @@ TextToCode.args = {
     },
   },
 };
+
```

Comment:
@ShivanshShalabh Here's an example of what you can do for your Storybook sample inputs

Reply:
Thanks, Alex! I will look into updating the storybook files too.